### PR TITLE
Train better weight model (by using LR find and 1cycle)

### PR DESCRIPTION
**Pull request by markus-hinsche**
https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/146

**Motivation:** our weight model was not very good, so we want to improve the accuracy.

**Solution:** 
- use LR find to find a good learning rate hyperparameter
- use 1cycle learning schedule to train

We found a promising model for weight! 🎉

**Changes:**
- We did not create a new experiment dir for weight, because only the config changed (this creates less code duplication)

# Details

weight val MAE before: 0.74kg
weight MAE now: 0.66kg ([q4-depthmap-plaincnn-weight-95k Run 12](https://ml.azure.com/experiments/id/7639b1a7-860f-4902-99b6-7b6e27cd9c2e/runs/q4-depthmap-plaincnn-weight-95k_1605774694_c216f6c5?wsid=/subscriptions/9b82ecea-6780-4b85-8acf-d27d79028f07/resourceGroups/cgm-ml-prod/providers/Microsoft.MachineLearningServices/workspaces/cgm-azureml-prod&tid=006dabd7-456d-465b-a87f-f7d557e319c8))

image

### CODE REVIEW

**Review by prashant905 — APPROVED** [review]
the changes seem to integrated properly.
